scripts/ai/AIDriveStrategyFieldWorkCourse.lua

```diff
@@ -67,7 +67,9 @@ function AIDriveStrategyFieldWorkCourse:start(course, startIx)
     if distance > 2 * self.turningRadius then
         self:debug('Start waypoint is far (%.1f m), use an alignment course to get there.', distance)
         self.course = course
-        self:startAlignmentTurn(course, startIx, 1)
+        --- Find the closest waypoint and create a alignment course form there to the start ix.
+        local _, _, ixClosestRightDirection, _ = course:getNearestWaypoints(self.vehicle:getAIDirectionNode())
+        self:startAlignmentTurn(course, ixClosestRightDirection, startIx)

```

Comment:
@pvaiko Do you think this is a good idea?

Reply:
Don't think is needed, `self:startAlignmentTurn(course, startIx, startIx)` should be fine, it'll start the alignment course at the vehicle's position anyway, see comment at startAlignmentTurn